templates/traefik/20/README.md

```diff
@@ -70,6 +70,7 @@ Use this labels if you choose extenal rancher integration.
 - traefik.backend.loadbalancer.method = < drr | wrr > # Override default lb algorithm `drr`
 - traefik.backend.circuitbreaker.expression = < expression > # Override default backend circuitbreaker expression `NetworkErrorRatio() > 0.5`
 - traefik.frontend.passHostHeader = < true | false > # Forward client Host header to the backend. Default `true`
+- traefik.frontend.rule = < rule > # Rule for host. Usually used to specify a host. I.e: 'Host:MySite.com'
```

Comment:
`traefik.frontend.rule` doesn't exist at `external` integration, just for `api` and `metadata`. For `external` integration use `traefik.domain`, `traefik.alias`,....